packages/Python/lldbsuite/test/lang/swift/dwarfimporter/C/TestSwiftDWARFImporterC.py

```diff
@@ -35,6 +35,7 @@ def build(self):
         self.assertTrue(os.path.isdir(include))
         shutil.rmtree(include)
 
+    @skipUnlessDarwin
```

Comment:
Nack. Is this a Linux specific issue or an architecture issue? Seems to pass on x86-64 linux.

Reply:
Hi @dcci,
This was done on the lines of "swift-lldb/packages/Python/lldbsuite/test/lang/swift/dwarfimporter/Objective-C/TestSwiftDWARFImporterObjC.py", which had a similar check - "@skipUnlessDarwin"
Nevertheless, I have made changes to check only for ppc64le for this test and also a few others.
Please review and approve the changes.
Thanks!

Reply:
There are tests that are legitimately not working on Linux, sometimes because they require Obj-C interoperability. That said, if the problem here is ppc64, then it's ppc64 the condition on which we skip.

Reply:
Sure. Thanks.